modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_view_display.node.branch.facility.yml

```diff
@@ -0,0 +1,93 @@
+langcode: en
```

Comment:
Fixed.
Code was moved after "classes" because we have similar logic for all template files.